shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -515,7 +572,7 @@ - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication* _Nonnu
                ICUDataPath:[fixtures stringByAppendingString:@"/icudtl.dat"]];
     FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"test"
                                                         project:project
-                                         allowHeadlessExecution:true];
+                                         allowHeadlessExecution:YES];
```

Comment:
Drive-by, noticed while comparing your test to my non-working test (now fixed).

Reply:
Ugh - thanks. I thought I'd done a sweep of these at one point but clearly missed a few. Will re-check today; maybe I never committed that branch?